vendor/github.com/rancher/types/config/context.go

```diff
@@ -465,9 +474,10 @@ func NewUserContext(scaledContext *ScaledContext, config rest.Config, clusterNam
 
 func (w *UserContext) Start(ctx context.Context) error {
 	logrus.Info("Starting cluster controllers for ", w.ClusterName)
-	controllers := w.Management.controllers()
-	controllers = append(controllers, w.controllers()...)
-	return controller.SyncThenStart(ctx, 5, controllers...)
+	if err := controller.SyncThenStart(w.runContext, 50, w.Management.controllers()...); err != nil {
```

Comment:
Curious to know what these magic numbers 50 and 5 are?

Reply:
threadiness i.e. how many goroutines to spin up to process work for those queues.